bin/setup_db

```diff
@@ -94,8 +99,8 @@ def setup_db(host, port, name):
         role_tables.for_each(r.db(name).table_create(r.row)).run(conn)
         role_tables.for_each(r.db(name).table(r.row).index_create('role_id')).run(conn)
 
-    except RqlRuntimeError:
-        LOGGER.info('Database already exists: %s', name)
+    except RqlRuntimeError as err:
```

Comment:
I believe this is ReqlRuntimeError according to RethinkDB's ReQL error types: 
https://www.rethinkdb.com/docs/error-types/

Reply:
I didn't modify that line, but that would be a great fix to make in another PR. Want to?